tutorial/bug_anim/assets/sprites/Fighter/index.json

```
@@ -0,0 +1,226 @@
+        },
+        {
+            "x": 0,
+            "y": 0,
+            "faceRight": 0,
+            "bitmapResolution": 2,
+            "name": "__animation_walk_walk-3",
+            "path": "__animation_walk_walk-3.png"
+        },
+        {
+            "x": 0,
+            "y": 0,
+            "faceRight": 0,
+            "bitmapResolution": 2,
+            "name": "__animation_walk_walk-4",
+            "path": "__animation_walk_walk-4.png"
+        },
+        {
+            "x": 0,
+            "y": 0,
+            "faceRight": 0,
+            "bitmapResolution": 2,
+            "name": "__animation_walk_walk-5",
+            "path": "__animation_walk_walk-5.png"
+        },
+        {
+            "x": 0,
+            "y": 0,
+            "faceRight": 0,
+            "bitmapResolution": 2,
+            "name": "__animation_walk_walk-6",
+            "path": "__animation_walk_walk-6.png"
+        },
+        {
+            "x": 0,
+            "y": 0,
+            "faceRight": 0,
+            "bitmapResolution": 2,
+            "name": "__animation_walk_walk-7",
+            "path": "__animation_walk_walk-7.png"
+        },
+        {
+            "x": 0,
+            "y": 0,
+            "faceRight": 0,
+            "bitmapResolution": 2,
+            "name": "__animation_walk_walk-8",
+            "path": "__animation_walk_walk-8.png"
+        },
+        {
+            "x": 0,
+            "y": 0,
+            "faceRight": 0,
+            "bitmapResolution": 2,
+            "name": "__animation_default_idle-1",
+            "path": "__animation_default_idle-1.png"
+        },
+        {
+            "x": 0,
+            "y": 0,
+            "faceRight": 0,
+            "bitmapResolution": 2,
+            "name": "__animation_default_idle-2",
+            "path": "__animation_default_idle-2.png"
+        },
+        {
+            "x": 0,
+            "y": 0,
+            "faceRight": 0,
+            "bitmapResolution": 2,
+            "name": "__animation_default_idle-3",
+            "path": "__animation_default_idle-3.png"
+        },
+        {
+            "x": 0,
+            "y": 0,
+            "faceRight": 0,
+            "bitmapResolution": 2,
+            "name": "__animation_default_idle-4",
+            "path": "__animation_default_idle-4.png"
+        },
+        {
+            "x": 0,
+            "y": 0,
+            "faceRight": 0,
+            "bitmapResolution": 2,
+            "name": "__animation_default_idle-5",
+            "path": "__animation_default_idle-5.png"
+        },
+        {
+            "x": 0,
+            "y": 0,
+            "faceRight": 0,
+            "bitmapResolution": 2,
+            "name": "__animation_default_idle-6",
+            "path": "__animation_default_idle-6.png"
+        }
+    ],
+    "fAnimations": {
+        "fight": {
+            "from": "__animation_fight_attack_1-1",
+            "to": "__animation_fight_attack_1-4",
+            "duration": 0.4,
```

If I set `frameFps` & `duration` at the same time:

```suggestion
            "frameFps": 1,
            "duration": 0.01,
```

it is expected that the `frameFps` controls the animation speed (one frame per second), and the `duration` does not take any effect on this animation (it takes effect only when bound to `step`)

While it does not work as expected now. It seems that the "fight" animation is played in 0.01 second, which is specified by `duration`.